app/Http/Livewire/SlackSettingsForm.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use Livewire\Component;
+use App\Models\Setting;
+
+class SlackSettingsForm extends Component
+{
+    public $slack_endpoint;
+    public $slack_channel;
+    public $slack_botname;
+    public $successMessage;
+
+    public Setting $setting;
+
+    protected $rules = [
```

Comment:
Does this duplicate the model-level validation that we have on the Settings model?